docs/dictionary/command/filter.lcdoc

```diff
@@ -111,17 +113,22 @@ You can match instances of special chars as follows:
 
 
 For example, the wildcard <filterPattern> `[[]A]*` will match any
-string beginning with `[A]`.
+string beginning with `[A]`. Broken down, there is `[[]` which equates
+to an open square bracket `[` followed by `A]*` as the closing square
+bracket is not a special character.
 
 The three bracketed forms can be combined to create more complex
 character classes, for example the pattern [!abcA-C] matches any
 character which is not a, b or c (upper or lower case)
 
-If no <filterPattern> type is specified, the <filterPattern> is handled
-as a wildcard.
+>**Note:** If the <paternType> `regex pattern` is specified, 
+the <filterPattern> evaluates to a <regular expression>.
+>
+>If no <patternType> is specified, the <filterPattern> is handled
+as a `wildcard pattern` type.
```

Comment:
Aha... I must have been looking at an older commit. I still think the block quoted note should _just_ highlight the no pattern type thing. Also I think the idea of splitting `<filterPattern>` into separate `<wildcardPattern>` and `<regularExpression>` parameters will help significantly. In that case you might change the note to:
```
If neither `wildcard pattern` or `regex pattern` are specified then the pattern is handled as a `wildcard pattern`.
```

Reply:
Perhaps `<regexPattern>` would be better to match `regex pattern` ?

Reply:
I like `<regexPattern>` to go with `<wildcardPattern>` as the two options unless you want to be more verbose with `<regexFilterPattern>` and `<wildcardFilterPattern>`.  I agree that since the syntax is now split into two forms, that the parameter should get unique names.

You could also refer to the syntax as the `regex pattern form` and `wildcard pattern form` throughout the document.

Here is another alternative for the note:
```
If the regex pattern form is used (`regex pattern`) then the filter pattern is a <regexPattern>.  
If the wildcard pattern form is used (`wildcard pattern` or the pattern is not specified) then the 
filter pattern is a <wildcardPattern>.
```

Reply:
Ohhh 😂I see what you mean now. I thought you were talking about my now redundant patternType. But you mean have, instead of filterPattern, these other two parameters listed. I’ll draft it up in the morning (uk time) for everyone to look at. Thanks Monte.

Reply:
@bwmilby, the verbose names in the already long syntax lines will likely be too long and unnecessary. The parameter description can probably make it clearer that they are filter patterns. 
I’ll look at the note with fresh eyes later and see if there is a simple approach that suits all of these ideas and bears in mind the upcoming changes to the command. Thanks Brian.